lambda-error-sns-sender/__init__.py,sha256=J4dNSO_Lah-WssTFLmdvY-wyv8rTX6k3TYfk7Kp1uQ8,27282
lambda-error-sns-sender/py.typed,sha256=AbpHGcgLb-kRsJGnwFEktk7uzpZOCcBY74-YBdrKVGs,1
lambda-error-sns-sender/_jsii/__init__.py,sha256=KooalMWSaoZA2Bo9eZqj4KlEYuXzoBBHi_Qr53dcB_c,428
lambda-error-sns-sender/_jsii/lambda-error-sns-sender@0.0.37.jsii.tgz,sha256=dlhvidstV3gBuRbfyIZI71oiR1sIMIHdC9R04d3D-0E,35425
lambda_error_sns_sender-0.0.37.dist-info/LICENSE,sha256=z8d0m5b2O9McPEK1xHG_dWgUBT6EfBDz6wA0F7xSPTA,11358
lambda_error_sns_sender-0.0.37.dist-info/METADATA,sha256=Vx3LUSoGwz3BwVOhUoSiHQ2L30TmsYBSFIMKc35UAe4,2102
lambda_error_sns_sender-0.0.37.dist-info/WHEEL,sha256=5sUXSg9e4bi7lTLOHcm6QEYwO5TIF1TNbTSVFVjcJcc,92
lambda_error_sns_sender-0.0.37.dist-info/top_level.txt,sha256=FkMA9jPLJsN588vpcg5IZxGjGTXyTWVbDBZNa2cu0yI,24
lambda_error_sns_sender-0.0.37.dist-info/RECORD,,
